apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -5246,6 +5251,15 @@
       "title": "Sorry, connection failed",
       "description": "Device detected but connection failed. Please try again or contact Ledger Support if the problem persists."
     },
+    "CasperInvalidTransferId": {
+      "title": "Transfer ID should be a positive number smaller than 18446744073709551615"
+    },
+    "InvalidMinimumAmount": {
+      "title": "Minimum amount to transfer is 2.5 CSPR"
+    },
+    "MayBlockAccount": {
+      "title": "Your account will be left with less than the 2.6 CSPR required to perform a send"
+    },
```

Comment:
For the 3 messages: instead of hardcoding the values in the translation, you can define them as constants in the code then provide them as a param of the translation key, using `{{ }}`,  see examples in other keys in this file.